src/eventreporter/DefaultMeetingEventReporter.ts

```diff
@@ -21,6 +21,7 @@ export default class DefaultMeetingEventReporter implements EventReporter, Destr
     'meetingStartFailed',
     'audioInputFailed',
     'videoInputFailed',
+    'meetingStartSucceeded',
```

Comment:
This does not seem related to retry with backoff. Should we document it?

Reply:
Yes, I have mentioned this in the description bullet point 5 :)